collect_app/src/main/res/layout/material_full_screen_dialog_toolbar.xml

```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<com.google.android.material.appbar.AppBarLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why did you remove this layout? Couldn't you just change that style here?

Reply:
I played around with moving it around to a shared module but in the end it seemed simpler to just have a shared style (`Widget.AndroidShared.Toolbar.Primary.CloseIcon`). I find `includes` can be a little awkward as you can't then customize/override attributes of the shared layout from where they are being used.